packages/client/src/internals/TRPCUntypedClient.ts

```diff
@@ -75,6 +75,7 @@ export interface TRPCSubscriptionObserver<TValue, TError> {
 export type CreateTRPCClientOptions<TRouter extends AnyRouter> =
   | CreateTRPCClientBaseOptions<TRouter> & {
       links: TRPCLink<TRouter>[];
+      abortOnUnmount?: boolean;
```

Comment:
Actually..... this doesn't belong in `@trpc/client` --- we don't want to have react-specific stuff there :)

Reply:
doh! good catch